src/AzDoBridge/Requests/LaunchRequestHandle.cs

```diff
@@ -20,7 +21,7 @@ public LaunchRequestHandle(ILogger log) : base(log)
             ILogger Log = log;
```

Comment:
this is a field, so please use the naming for fields. Please also add the protection level.